t/errlog.t

```diff
@@ -9,21 +9,32 @@ log_level('error');
 
 repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 2 + 11);
+plan tests => repeat_each() * (blocks() * 2 + 12);
 
 my $pwd = cwd();
 
 add_block_preprocessor(sub {
     my $block = shift;
 
+    my $test_name = $block->name();
     my $http_config = $block->http_config || '';
-    $http_config .= <<_EOC_;
 
-    lua_package_path "$pwd/lib/?.lua;../lua-resty-lrucache/lib/?.lua;;";
-    init_by_lua_block {
-        require "resty.core"
+    if (index($test_name, "TEST 25: ") != -1) {
```

Comment:
I don't have better way  :(

A note: the test name is unstable, if some guy added a test case before "TEST 25", this case will not work as we expect. 

here is an example: https://github.com/openresty/lua-nginx-module/blob/master/t/134-worker-count-5.t